test/CreatorToken.t.sol

```diff
@@ -49,6 +49,44 @@ contract CreatorTokenTest is Test {
     (uint256 _creatorFee, uint256 _adminFee) = creatorToken.calculateFees(_price);
     return _price + _creatorFee + _adminFee;
   }
+
+  function buyAToken(address _buyer) public {
+    (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextBuyPrice();
+    uint256 _totalPrice = _tokenPrice + _creatorFee + _adminFee;
+    uint256 originalCreatorTokenBalance = payToken.balanceOf(address(creatorToken));
+    uint256 originalBuyerBalance = creatorToken.balanceOf(_buyer);
+    uint256 originalTotalSupply = creatorToken.totalSupply();
+    deal(address(payToken), _buyer, _totalPrice);
+
+    vm.startPrank(_buyer);
+    payToken.approve(address(creatorToken), type(uint256).max);
+    creatorToken.buy(_totalPrice);
+    vm.stopPrank();
+
+    assertEq(creatorToken.balanceOf(_buyer), originalBuyerBalance + 1);
+    assertEq(creatorToken.ownerOf(creatorToken.lastId()), _buyer);
+    assertEq(payToken.balanceOf(address(creatorToken)), originalCreatorTokenBalance + _tokenPrice);
+    assertEq(creatorToken.totalSupply(), originalTotalSupply + 1);
+  }
+
+  function sellAToken(address _seller, uint256 _tokenId) public {
+    require(creatorToken.balanceOf(_seller) > 0, "seller does not own any tokens");
+    require(creatorToken.ownerOf(_tokenId) == _seller, "seller does not own the token");
+    (uint256 _tokenPrice, uint256 _creatorFee, uint256 _adminFee) = creatorToken.nextSellPrice();
+    uint256 _netProceeds = _tokenPrice - _creatorFee - _adminFee;
+    uint256 originalCreatorTokenBalance = payToken.balanceOf(address(creatorToken));
+    uint256 originalSellerBalance = creatorToken.balanceOf(_seller);
+    uint256 originalTotalSupply = creatorToken.totalSupply();
```

Comment:
Similar comments about these variable names, etc... as on the buy side

Reply:
Updated this in the same commit as rest of the naming updates: https://github.com/showtime-xyz/creator-tokens/pull/14/commits/89f1666530442849e5fcc6a35261447ee2cf1cbf